juriscraper/pacer/email.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+import re
+
+from ..lib.string_utils import clean_string, convert_date_string, harmonize
+from .reports import BaseReport
+
+
+class NotificationEmail(BaseReport):
+    def __init__(self, court_id):
+        self.court_id = court_id
+
+    @property
+    def data(self):
+        base = {
+            "court_id": self.court_id,
+        }
+        parsed = {}
+        if self.tree is not None:
+            parsed = {
+                "case_name": self._get_case_name(),
+                "docket_number": self._get_docket_number(),
+                "date_filed": self._get_date_filed(),
+            }
+
+        return {**(base or {}), **(parsed or {})}
+
+    def _get_case_name(self):
+        try:
+            path = '//td[contains(., "Case Name:")]/following-sibling::td[1]/text()'
+            case_name = harmonize(self.tree.xpath(path)[0])
```

Comment:
@mlissner To point 4. Thanks for the tip. Went with harmonize for the case name, clean_string for the case_number and the convert_date_string for the date. All worked very well.